### DISCUSSION TITLE
On disconnect a remote system switches to 640x480 screen resolution

### DISCUSSION THREAD

**Discussion by Glenn1963** [rustdesk/rustdesk#5058] [General]
I have one local headless system (direct connect) that, upon disconnecting my Rustdesk client, reverts to 640x480 video.

Then Rustdesk will not reconnect until I switch that system's video back to 1920x1080, which means I have to connect a display and a mouse every time.

I could deal with the video glitch if I could connect, but somehow that gets broken too.

Are there any settings I should be using to try to prevent this behavior?

Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/4974

**Reply by Glenn1963** [reply]
Oops, thanks for that :) I did search but clearly didn't know what to search for, lol. 
I appreciate the heads-up.
